README.md

```diff
@@ -1,24 +1,80 @@
-# README
+## Reformer
+The reformer is a user creation app. The app creates a user after a form has been filled by the particular user.
+##### Prerequisites
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The setups steps expect following tools installed on the system.
 
-Things you may want to cover:
+- Github
+- Ruby [3.0.1](https://github.com/ruby/ruby/tree/ruby_3_1)
+- Rails [7.0.1](https://github.com/rails/rails/tree/v7.0.1)
 
-* Ruby version
+## 1. Check out the repository
+```shell
+git clone [git@github.com:YinkTech/re-former.git](https://github.com/YinkTech/re-former)
+cd micro-reddit
```

Comment:
- [ ] Would like to see everyone contribute and commit to the changes requested.